rule/repository_memory.go

```diff
@@ -111,7 +111,7 @@ func (m *RepositoryMemory) Set(ctx context.Context, rules []Rule) error {
 	for _, check := range rules {
 		if err := m.r.RuleValidator().Validate(&check); err != nil {
 			viperx.LoggerWithValidationErrorFields(m.r.Logger(), err).WithError(err).
-				Errorf("A rule uses a malformed configuration and all URLs matching this rule will not work. You should resolve this issue now.")
+				Errorf("Rule %s uses a malformed configuration and all URLs matching this rule will not work. You should resolve this issue now.", check.ID)
```

Comment:
We can actually include the full rule here :)

```suggestion
				WithField("rule_id", check.ID).
				WithField("rule", check).
				Errorf("A rule uses a malformed configuration and all URLs matching this rule will not work. You should resolve this issue now.", check.ID)
```

What do you think?

Reply:
looks good in the code, but it will print out bytes.

`ERRO[0000] Rule chatbot-webhook uses a malformed configuration and all URLs matching this rule will not work. You should resolve this issue now.  error="authenticator matching this route is misconfigured or disabled" rule="{chatbot-webhook   0xc0004c20c0 [{oauth2_client_credentials []}] {keto_engine_acp_ory [123 10 32 32 32 32 32 32 32 32 34 114 101 113 117 105 114 101 100 95 97 99 116 105 111 110 34 58 32 34 99 97 108 108 34 44 10 32 32 32 32 32 32 32 32 34 114 101 113 117 105 114 101 100 95 114 101 115 111 117 114 99 101 34 58 32 34 99 104 97 116 98 111 116 58 119 101 98 104 111 111 107 34 10 32 32 32 32 32 32 125]} [{noop []}] [] {false  } <nil>}" rule_id=chatbot-webhook
`
Maybe just the rule_id field then?

Reply:
Yeah, makes sense!